server/pathfinding/initializeCollisions.js

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
removed files from server/pathfinding . I'm keeping the folder in case we refactor monster pathfinding to execute on the server.